### DISCUSSION TITLE
version 7.7 lost resp_code field

### DISCUSSION THREAD

**Discussion by lovelypp777** [cesanta/mongoose#1646] [Ask questions, get answers] [answered]
I use version 7.7 
I find in structure mg_http_message the  resp_code field is lost. But in old version it exists.

I don't know how to fetch the http resp code then.
thanks

**Comment by scaprile** [comment, marked as answer]
Did you try calling [mg_http_status(hm)](https://mongoose.ws/documentation/#mg_http_status)  ?
